docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Prerequisites from '../_partials/_prerequisites-sdk-api-key.md';
+import Steps from '../_partials/_get-started-steps.md';
+import ConnectToGDN from '../_partials/_connect-to-gdn-code-block.md';
+
+This article is an introduction to using stream workers with the Macrometa GDN console.
+
+<Prerequisites />
+
+## Get Started with Stream Workers
+
+This page guides you through creating a stream worker and updating it.
+
+<Steps />
+
+If you want to skip the explanation and just run the code, then go directly to the [Full Demo File](#full-demo-file).
+
+### Step 1. Connect to GDN
+
+To use streams with Macrometa Global Data Network (GDN), you must first establish a connection to a local region.
+
+<ConnectToGDN />
+
+### Step 2. Define Query Name
+
+Write a name and description. If no `qlVersion` is specified, default is `2`. Do not change this value unless you are working with a version 1 stream.
+
+```sql
+@App:name('sample-cargo-app')
+@App:description('Basic Stream application to demonstrate reading data from input stream and store it in the collection. The stream & collection will be created automatically if they do not already exist.')
+@App:qlVersion('2')
+```
+
+### Step 3. Create Source
+
+Example to create a source for a stream worker.
+
+```sql
+-- Defines `SampleCargoAppInputTable` Source.
+CREATE SOURCE SampleCargoAppInputTable WITH (type = 'database', collection = "SampleCargoAppInputTable", collection.type="doc" , replication.type="global", map.type='json') (weight int);
+```
+
+### Step 4. Create Sink
+
+Example to create a sink for a stream worker.
+
+```sql
+-- Defines `SampleCargoAppDestStream` Stream.
+CREATE SINK STREAM SampleCargoAppDestStream (weight int);
+```
+
+### Step 5. Write and Save Query
+
+Example query that adds data from the source into the sink.
+
+```sql
+-- Data Processing
+@info(name='Query')
+INSERT INTO SampleCargoAppDestStream
+SELECT weight
+FROM SampleCargoAppInputTable;
+```
+
+### Step 6. Add Data to the Source
+
+Upload the following data to the source you created:
+
+```sql
+{"weight": 1}
+{"weight": 2}
+{"weight": 3}
+{"weight": 4}
+{"weight": 5}
+```
+
+When you view the output for `SampleCargoAppDestStream`, you will see results similar to the following:
```

Comment:
How do I view the output?

Reply:
@diana-macrometa View it on the stream console in the GUI.  Is there a better way to phrase this?

"View the stream output in the GUI for `SampleCargoAppDestStream`. You will see results similar to the following:"